src/main/java/io/confluent/connect/hdfs/TopicPartitionWriter.java

```diff
@@ -295,15 +297,23 @@ public void close() throws ConnectException {
           commitFile(encodedPartition);
         }
       } catch (IOException e) {
-        log.error("Error rotating {} when closing TopicPartitionWriter.", tempFiles.get(encodedPartition));
+        log.error("Error rotating {} when closing TopicPartitionWriter.",
+                  tempFiles.get(encodedPartition));
+        ByteArrayOutputStream baos = new ByteArrayOutputStream();
```

Comment:
Is all this necessary? Why not just use this form?

```
log.error("Error rotating {} when closing TopicPartitionWriter.",
+                  tempFiles.get(encodedPartition), e);
```

i.e. just add the exception to the end of the original log message, which should dump its message + stacktrace to the log.